credsweeper/secret/config.json

```diff
@@ -1,150 +1,1097 @@
+      ".ear",
+      ".eot",
+      ".exe",
+      ".gif",
+      ".gz",
+      ".ico",
+      ".img",
+      ".info",
+      ".jar",
+      ".jpeg",
+      ".jpg",
+      ".m4a",
+      ".mat",
+      ".mo",
+      ".mp3",
+      ".mp4",
+      ".npy",
+      ".npz",
+      ".ogg",
+      ".pak",
+      ".pdf",
+      ".png",
+      ".pptx",
+      ".pyc",
+      ".pyd",
+      ".pyo",
+      ".rar",
+      ".realm",
+      ".rst",
+      ".s7z",
+      ".scss",
+      ".so",
+      ".sum",
+      ".svg",
+      ".tar",
+      ".tif",
+      ".tiff",
+      ".ttf",
+      ".vcxproj",
+      ".war",
+      ".wav",
+      ".webm",
+      ".webp",
+      ".woff",
+      ".xls",
+      ".xlsx",
+      ".yuv",
+      ".zip"
     ],
-    "source_quote_ext": [
-        ".go",
-        ".ipynb",
-        ".ts",
-        ".java",
-        ".js",
-        ".py",
-        ".cpp",
-        ".c",
-        ".h",
-        ".hpp"
-    ],
-    "find_by_ext_list": [
-        ".pem",
-        ".crt",
-        ".cer",
-        ".csr",
-        ".der",
-        ".pfx",
-        ".p12",
-        ".key",
-        ".jks"
-    ],
-    "check_for_literals": true,
-    "min_pattern_value_length": 12,
-    "min_keyword_value_length": 4,
-    "line_data_output": [
-        "line",
-        "line_num",
-        "path",
-        "value",
-        "variable",
-        "entropy_validation"
-    ],
-    "candidate_output": [
-        "rule",
-        "severity",
-        "line_data_list",
-        "api_validation",
-        "ml_validation",
-        "ml_probability"
+    "path": [
+      "/.git/",
+      "/.idea/",
+      "/.svn/",
+      "/__pycache__/",
+      "/node_modules/",
+      "/target/",
+      "/venv/"
     ]
+  },
+  "source_ext": [
+    ".aspx",
+    ".cs",
+    ".cshtml",
+    ".ejs",
+    ".erb",
+    ".go",
+    ".html",
+    ".ipynb",
+    ".jsp",
+    ".jsx",
+    ".php",
+    ".phtml",
+    ".rb",
+    ".sh",
+    ".swift",
+    ".ts",
+    ".twig",
+    ".vue",
+    ".xhtml",
+    ".java",
+    ".js",
+    ".py",
+    ".cpp",
+    ".c",
+    ".h",
+    ".hpp",
+    ".mm",
+    ".cu",
+    ".y",
+    ".vb",
+    ".m",
+    ".cu"
+  ],
+  "source_quote_ext": [
+    ".go",
+    ".ipynb",
+    ".ts",
+    ".java",
+    ".js",
+    ".py",
+    ".cpp",
+    ".c",
+    ".h",
+    ".hpp"
+  ],
+  "find_by_ext_list": [
+    ".pem",
+    ".crt",
+    ".cer",
+    ".csr",
+    ".der",
+    ".pfx",
+    ".p12",
+    ".key",
+    ".jks"
+  ],
+  "check_for_literals": true,
+  "min_pattern_value_length": 12,
+  "min_keyword_value_length": 4,
+  "line_data_output": [
+    "line",
+    "line_num",
+    "path",
+    "value",
+    "variable",
+    "entropy_validation"
+  ],
+  "candidate_output": [
+    "rule",
+    "severity",
+    "line_data_list",
+    "api_validation",
+    "ml_validation",
+    "ml_probability"
+  ],
+  "keyword_checklist": [
+    "1234",
+    "abort",
+    "about",
+    "above",
+    "absolute",
+    "abstract",
+    "accent",
+    "accept",
```

Comment:
Adding `keyword_checklist` makes `config.json` file too long.. isn't it?
This is main reason that I created the `txt` file for it.

Reply:
And I want to remind that it's more difficult to add values to json than plaintext

I think all blacklists I remember always was stored as a plaintext, here few examples: 
https://github.com/danielmiessler/SecLists/tree/master/Passwords
https://github.com/berandal666/Passwords
https://github.com/Python-Fa/Password-List

Reply:
Ok. May i remove only keyword_checklist.py and use simply logit to load the txt file?
And could you review draft https://github.com/Samsung/CredSweeper/pull/205 ? - the issue i found when working with the checklist.,